pkg/regression/v2_volumes.go

```diff
@@ -85,13 +86,43 @@ var _ = DescribeRegression("Volume Testing (v2)", func(tc *TestContext) {
 			Expect(lun).ToNot(Equal(0))
 		})
 
-		It("should be able to unmap volume", func() {
+		It("should successfully create a second volume", func() {
+
+			logger := klog.FromContext(tc.Config.Ctx)
+			volume, status, err := client.CreateVolume(volname2, size, tc.Config.StorageController.Pool, poolType)
+			logger.V(3).Info("CreateVolume", "name", volname2, "size", size, "wwn", volume.Wwn, "mc-response", status.ResponseTypeNumeric)
+
+			Expect(err).To(BeNil())
+			Expect(status.ResponseTypeNumeric).To(Equal(storageapi.ApiSuccess))
+			Expect(volume.Wwn).ShouldNot(BeEmpty())
+
+			// Dump volumes
+			response, status, err := client.ShowVolumes(volname1)
+			Expect(err).To(BeNil())
+			Expect(status.ResponseTypeNumeric).To(Equal(storageapi.ApiSuccess))
+			ShowVolumes(logger, response)
+		})
+
+		It("should be able to publish/map a second volume", func() {
+
+			logger := klog.FromContext(tc.Config.Ctx)
+			lun, err := client.PublishVolume(volname2, tc.Config.StorageController.Initiator)
+			logger.V(3).Info("PublishVolume", "name", volname1, "lun", lun)
```

Comment:
Should this be logger.V(3).Info("PublishVolume", "name", volname2, "lun", lun)

Reply:
Fixed in commit 4a29dad